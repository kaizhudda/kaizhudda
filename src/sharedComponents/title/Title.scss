@import '../../styles/variables';
@import '../../styles/media-queries';

.Title {
  font-family: $font-body;
  font-weight: 300;
  font-size: 3rem;
  margin-bottom: 10px;
  color: #22a39f;

  @include phone-only {
    font-size: 2rem;
  }
}