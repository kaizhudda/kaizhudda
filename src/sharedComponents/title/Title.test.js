import React from 'react';
import { shallow } from 'enzyme';
import Title from './index';

const title = 'Test Title';
const props = {
  className: 'title'
};
let wrapped = shallow(<Title {...props}>{title}</Title>); 

describe('Title', () => {
  it('should render the Title Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('renders the Titles children', () => {
    expect(wrapped.find('h1').text()).toEqual(title);
  });

  it('renders the Class Name on Title Component', () => {
    expect(wrapped.find('h1').hasClass(props.className)).toBe(true)
  });
});

describe('Title with no ClassName', () => {
  beforeEach(() => {
    wrapped = shallow(<Title>{title}</Title>);
  });

  it('renders the Class Name on Title Component', () => {
    expect(wrapped.find('h1').hasClass(props.className)).toBe(false)
  });
});