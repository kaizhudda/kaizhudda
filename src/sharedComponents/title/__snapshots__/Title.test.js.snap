// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Title Component correctly 1`] = `
<h1
  className="Title title"
>
  Test Title
</h1>
`;
